### PR TITLE
Blurt Integration - Part 1

### PR DESCRIPTION
**Pull request by eastmaels**
Blurt is a graphene based blockchain forked from STEEM.  Two of its differentiating factor from its sister chains STEEM and HIVE is that Blurt doesn't have downvotes and no stable coin. It adds an alternative for users to choose as platform to create their posts.

This PR aims to add support to simultaneously (technically serially) post one's DTube video to https://blurt.blog if one logs his Blurt account.

As advised before by @heimindanger and @techcoderx, integrating DTube with Blurt requires mainly two things:

1. Add Blurt as a sign-in option
2. Make sure that d.tube successfully transacts with the chain

For 1, I added two [loginblurt.html](client/views/pages/login/loginblurt/loginblurt.html) and [loginblurt.js](client/views/pages/login/loginblurt/loginblurt.js).
For 2, it required a modification to [broadcast.js](client/broadcast.js).

Demo: https://d.tube/#!/v/eastmael/y41nwycmohh

For more description/details on Blurt blockchain, please refer to Blurt's official repository - https://gitlab.com/blurt/blurt.

### CODE REVIEW

**Comment by brishtiteveja** [comment]
Please write some descriptions explaining the motivation and implementation.

File: client/collections/videos.js [hunk 1]
```diff
@@ -916,6 +917,129 @@ Videos.parseFromSteem = function(video, isComment, network) {
...
+      if (newVideo.json.info && newVideo.json.content)
+          newVideo.json = Videos.convertToNewFormat(newVideo.json, video)
+      newVideo.json.app = JSON.parse(video.json_metadata).app
+      newVideo.json.tags = JSON.parse(video.json_metadata).tags
+      if (!newVideo.json.videoId && !newVideo.json.files && !newVideo.json.ipfs.snaphash)
+          return
+  } catch (e) {
+      return
+  }
+  if (!isComment && !newVideo) return
```
Comment (brishtiteveja) [inline client/collections/videos.js:957]: fix indentation in this if/else block

**Review by brishtiteveja — CHANGES_REQUESTED** [review]
Looks pretty okay to me other than some integration issue.

File: client/collections/videos.js (hunk 1 above)
Comment (eastmaels) [reply client/collections/videos.js]: Thanks for the review. Fixed indention.

**Comment by tibfox** [comment]
are thise requested changes still open or can we finally merge that?

**Comment by eastmaels** [comment]
> are thise requested changes still open or can we finally merge that?

I've already resolved the requested changes and have pushed additional changes requested by @d00k13 via discord.

**Review by techcoderx — APPROVED** [review]

**Comment by brishtiteveja** [comment]
If this is approved by @techcoderx, may be we can merge.

**Pull request merged**